docker/opbeans/java/Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM opbeans/opbeans-java:latest
+FROM maven:3.5.3-jdk-10
+# setting --build-arg JAVA_AGENT_BRANCH=<branch> causes the agent to be built from source
+# instead of using the agent which comes pre-built with opbeans-java:latest
+ARG JAVA_AGENT_REPO=elastic/apm-agent-java
+ARG JAVA_AGENT_BRANCH=
```

Comment:
This is currently always set by `compose.py`, I think changing `"master"` to `""` should be good enough.

https://github.com/elastic/apm-integration-testing/blob/b0cef3414173aa3e5ec8100d1e2ec2b0d810f504/scripts/compose.py#L1730

Reply:
done